Penteract/Assets/Scripts/Fang.cpp

```diff
@@ -184,7 +189,7 @@ void Fang::Dash() {
 		newPosition += dashSpeed * dashDirection;
 		agent->SetMoveTarget(newPosition, false);
 	} else {
-		if (hasDashed)trailDelay();
+		if (hasDashed) trailDelay();
```

Comment:
I thought you would put it in capital letters, in my version it is already set, you're welcome ;)